src/ui.h

```diff
@@ -430,7 +429,6 @@ class TextWindow {
     static void ScreenChangeShowContourAreas(int link, uint32_t v);
     static void ScreenChangeCheckClosedContour(int link, uint32_t v);
     static void ScreenChangeTurntableNav(int link, uint32_t v);
-    static void ScreenChangeImmediatelyEditDimension(int link, uint32_t v);
```

Comment:
Not sure why this was removed - its uses were not removed resulting in build issues.